.github/workflows/build_pytorch_yolo.yml

```diff
@@ -0,0 +1,69 @@
+name: Pytorch Yolo Detection
```

Comment:
@Charles-Schleich I think the workflow should be added to the .github/workflows/pytorch.yml workflow file because that contains all the examples. 
Let me know what do you think @juntao

Reply:
I think it is okay to have it as a separate flow since YOLO is a widely used model that has its own release cycles.